GOVERNANCE.md

```diff
@@ -0,0 +1,81 @@
+to date.
+
+[marthas-rules]: https://journals.sagepub.com/doi/10.1177/088610998600100206
+
+## Making Decisions
+
+1.  Before each meeting, anyone who wishes may sponsor a proposal by filing an
+    issue in the GitHub repository tagged "proposal".  Proposals must be filed
+    at least 24 hours before a meeting in order to be considered at that
+    meeting, and must include:
+    -   a one-line summary (the subject line of the issue)
+    -   the full text of the proposal
+    -   any required background information
+    -   pros and cons
+    -   possible alternatives
+
+2.  A quorum is established in a meeting if half or more of voting members are
+    present.
+
+3.  Once a person has sponsored a proposal, they are responsible for it.  The
+    group may not discuss or vote on the issue unless the sponsor or their
+    delegate is present.  The sponsor is also responsible for presenting the
+    item to the group.
+
+4.  After the sponsor presents the proposal, a sense vote is cast for the
+    proposal prior to any discussion:
+    -   Who likes the proposal?
+    -   Who can live with the proposal?
+    -   Who is uncomfortable with the proposal?
+
+5.  If all of the group likes or can live with the proposal, it is immediately
+    accepted with no further discussion.
+
+6.  If most of the group is uncomfortable with the proposal, it is postponed for
+    further rework by the sponsor.
+
+7.  If any members are uncomfortable they can briefly state their objections.  A
+    timer is then set for a brief discussion moderated by the facilitator.
+    After 10 minutes or when no one has anything further to add (whichever comes
+    first), the facilitator calls for a yes-or-no vote on the question: "Should
+    we implement this decision over the stated objections?"  If a majority votes
+    "yes" the proposal is implemented.  Otherwise, the proposal is returned to
+    the sponsor for further work.
+
+## Suffrage
+
+1.  The following people are currently voting members of this project:
+    -   [Ian Flores Siaca](https://audassimus.com/)
+    -   [Zhian Kamvar](https://zkamvar.netlify.app/)
+    -   [François Michonneau](https://francoismichonneau.net/)
+    -   [Greg Wilson](http://third-bit.com/)
+
+2.  To become a voting member:
+    1.  A current member must sponsor a proposal to add the candidate.
+    2.  That proposal must pass as outlined in the "Making Decisions" section of
+        this page.
+
+3.  Voting members may resign from the project at any time, or lose their vote
+    if a proposal to that effect is passed as outlined in the "Making Decisions"
+    section of this page.
+    1.  A proposal to remove someone's vote may not contain any other business.
+    2.  Each person must be the subject of a separate proposal.
+    3.  The person in question has the right to vote on the proposal to remove
+        their vote.
+
+## Recording Decisions
+
+Proposals that have been accepted are listed below in reverse chronological
```

Comment:
Do we want to list them here? How to keep track of it? Manual?

Reply:
Yes and yes until the group decides on something else, such as a wiki page in the GitHub repo or a separate `RESOLUTIONS.md` file (both of which feel like overkill for now).